application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -238,6 +239,7 @@ require(['jquery'], function ($) {
                   targetPage  : (targetPage? targetPage: ''),
                   targetAction: (targetAction? targetAction: ''),
                   queryString : (queryString? queryString: ''),
+                  reflex      : (reflex? reflex: 0),
```

Comment:
I followed what was done for `backdrop`: https://github.com/xwiki-contrib/application-tour/pull/15/files/e963c21125a832877efff88264e6ab55b569e0e9#diff-f27b27cae7d2f1dbadbb5be908f04950R236